agenta-backend/pyproject.toml

```diff
@@ -32,14 +32,16 @@ watchdog = {extras = ["watchmedo"], version = "^3.0.0"}
 beanie = "^1.25.0"
 newrelic = "^9.8.0"
 aioboto3 = "^12.4.0"
-ragas = "^0.1.8"
+ragas = ">=0.1.8"
 openai = "^1.30.4"
 sqlalchemy = "^2.0.30"
 asyncpg = "^0.29.0"
 psycopg2-binary = "^2.9.9"
 uuid-utils = "^0.7.0"
 sqlalchemy-json = "^0.7.0"
 tqdm = "^4.66.4"
+langchain-community = "^0.2.9"
```

Comment:
@aybruhm where this is used ?

Reply:
Not used anymore. I've removed [it](https://github.com/Agenta-AI/agenta/pull/1921/commits/0eec9dc5f09a7af1cb3c3263ccb971aaa33f194c).

Reply:
thanks @aybruhm